src/lib/KeyShares/KeyShares.ts

```diff
@@ -71,6 +67,42 @@ export class KeyShares {
     return this.payload;
   }
 
+  generateKeySharesFromBytes(shares: string, operatorIds: any[]): any {
+    const operatorCount = operatorIds.length;
+    shares = shares.replace('0x', '');
+    const pkLength = parseInt(shares.substring(0, 4), 16);
+
+    // get the public keys part
+    const pkSplit = shares.substring(4, pkLength + 2);
+    const pkArray = ethers.utils.arrayify('0x' + pkSplit);
+    const sharesPublicKeys = this.splitArray(operatorCount, pkArray).map(item =>
+      ethers.utils.hexlify(item),
+    );
+
+    const eSplit = shares.substring(pkLength + 2);
+    const eArray = ethers.utils.arrayify('0x' + eSplit);
+    const encryptedKeys = this.splitArray(operatorCount, eArray).map(item =>
+      Buffer.from(ethers.utils.hexlify(item).replace('0x', ''), 'hex').toString(
+        'base64',
+      ),
+    );
+    return {
+      sharesPublicKeys,
+      encryptedKeys,
+    };
+  }
```

Comment:
Commented about it here:

https://github.com/bloxapp/ssv-keys/pull/34#discussion_r1182317520

Btw, why 2 prs have the same changes set?